tests/raylib-umka-tests.um

```diff
@@ -36,6 +41,13 @@ fn main() {
     Assert.True(expectedRect.width == rect3.width, "GetCollisionRec().width")
     Assert.True(expectedRect.height == rect3.height, "GetCollisionRec().height")
 
+    raylib.TraceLog(raylib.LOG_INFO, "1")
+    raylib.SetLoadFileTextCallback(loadText)
+    raylib.TraceLog(raylib.LOG_INFO, "2")
+    var output:str = raylib.LoadFileText("Assert.um")
+    raylib.TraceLog(raylib.LOG_INFO, "3")
+    raylib.TraceLog(raylib.LOG_INFO, output)
```

Comment:
Getting a segfault here. Likely need to translate the callbacks into umka function calls?